pennylane/tape/circuit_graph.py

```diff
@@ -48,6 +53,23 @@ def observables(self):
         """Observables in the circuit."""
         return self._observables
 
+    @property
+    def depth(self):
+        """Depth of the quantum circuit (longest path in the DAG)."""
+        # If there are no operations in the circuit, the depth is 0
+        if len(self.operations) == 0:
+            self._depth = 0
```

Comment:
Alternatively, you could do
```suggestion
        if not self.operations:
            self._depth = 0
```

Pylint tends to prefer this, so I'm surprised it's not complaining 🤔 `len(something) == 0` is generally avoided, since it contains a redundant boolean comparison